Source/SwiftLintFramework/Rules/RuleConfigurations/BlanketDisableCommandConfiguration.swift

```diff
@@ -0,0 +1,33 @@
+public struct BlanketDisableCommandConfiguration: SeverityBasedRuleConfiguration, Equatable {
+    public private(set) var severityConfiguration = SeverityConfiguration(.warning)
+    public private(set) var allowedRuleIdentifiers: Set<String> = ["file_length", "single_test_class"]
+    public private(set) var alwaysBlanketDisableRuleIdentifiers: Set<String> = []
```

Comment:
Isn't it the other way around?
```suggestion
    public private(set) var allowedRuleIdentifiers: Set<String> = []
    public private(set) var alwaysBlanketDisableRuleIdentifiers: Set<String> = ["file_length", "single_test_class"]
```

Reply:
This is actually correct - the intention is that by default, the rule will not complain about 

`// swiftlint:disable file_length`, for example, even though it's not re-enabled, because it makes sense to just disable it once for the whole file, and there's no point in doing `// swiftlint:enable file_length` at the end of the file.

Adding a rule to the `alwaysBlanketDisableRuleIdentifiers` goes one step further, by warning the user on any SwiftLint command affecting the rule, that is anything other than `// swiftlint:disable file_length`. I didn't want to enable that by default.

Reply:
I have made one slight modification though - any rules in `alwaysBlanketDisableRuleIdentifiers` are automatically "allowed"

Reply:
Hm, got it ... this is tricky, though.

Reply:
I think this is the right behaviour, but maybe the naming isn't quite perfect. I went with `allowed_rules` and `always_blanket_disable`, but maybe there's something better, although I don't have any good suggestions.

I've also just added `file_header`, `file_name`, and `file_name_no_space` to the default allowed rules list.